runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/getConstantPool/gcp_bcdump.c

```diff
@@ -282,7 +282,7 @@ const char * const maciek_JavaBCNames[] = {
 "JBunimplemented" /* 240 */,
 "JBunimplemented" /* 241 */,
 "JBunimplemented" /* 242 */,
-"JBreturnToMicroJIT" /* 243 */,
+"JBunimplemented" /* 243 */,
```

Comment:
I think this change should be conditional on `defined(J9VM_OPT_MICROJIT)`.

Reply:
I've replaced it with an `if defined() else endif` section. This might need to be removed later though; as far as I know `JBreturnToMicroJIT` was for the old MicroJIT and nothing is using it anymore that I know of.